.github/workflows/assemble.yml

```
@@ -0,0 +1,24 @@
+name: Gradle Assemble
+on: [pull_request]
+
+jobs:
+  assemble:
+    if: github.repository == 'opensearch-project/OpenSearch'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        java: [ 11 ]
```

I believe we maintain JDK-8 compatibility in 1.x:

```suggestion
        java: [ 8, 11 ]
```